recipes/tabulate/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "tabulate" %}
+{% set version = " v1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/p-ranav/tabulate/archive/{{ version }}.tar.gz
+  sha256: 34eda3605e2ce0c7ca339fa2af6a0c77a5e49321b2f549ff5313f955273c8ac8
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    run_exports:
+      - {{ pin_subpackage(name, max_pin='x.x') }}
+  host:
```

Comment:
tabulate shouldn't be in your `host` section. in fact, if tabulate doesn't require any other packages, you probably don't need a host section at all :)

Reply:
> Oh, one more thing that I got confused...
> Tabulate is using some libraries to visualization like [wchar.h](https://github.com/p-ranav/tabulate/blob/ff9dc4cdfa5c522acb15e87f8608031e63cdf619/include/tabulate/utf8.hpp#L43) and some for colors depending on your OS, like [these](https://github.com/p-ranav/tabulate/blob/ff9dc4cdfa5c522acb15e87f8608031e63cdf619/include/tabulate/termcolor.hpp#L32-L37). How do I deal with this kind of thing?
> Thaaaankkksssss

This is related to what I asked here, I think.
I'm not sure if I should include this libs here.